test/e2e/specs/preview.test.js

```diff
@@ -94,11 +99,14 @@ describe( 'Preview', () => {
 		await editorPage.bringToFront();
 		await publishPost();
 		await Promise.all( [
-			page.waitForFunction( () => ! document.querySelector( '.editor-post-preview' ) ),
-			page.click( '.editor-post-publish-panel__header button' ),
+			editorPage.waitForFunction( () => ! document.querySelector( '.editor-post-preview' ) ),
+			editorPage.click( '.editor-post-publish-panel__header button' ),
 		] );
 		expectedPreviewURL = await editorPage.$eval( '.notice-success a', ( node ) => node.href );
-		await waitForPreviewNavigation( previewPage );
+		// At this point, the preview page can lose its name, so we close it and open a fresh one
```

Comment:
What's meant by "lose its name" ? I don't know that this is relevant any longer after #8330 . There should only ever be one preview window.

Reply:
At this point in the test, the existing preview window's `window.name` can be blank, and so a second one opens. I ran the test in non-headless mode and double checked. It might not happen always, but it happened enough for me to have to do this to get the test to pass consistently.

Reply:
I would be more curious in understanding why the preview name becomes blank, as that sounds like a bug in and of itself.

Reply:
Oh, agreed totally :) Would you be ok if we open an issue and addressed it in another PR? The e2e intermittent test failures have blocked us recently. I'd be happy to start investigating immediately.

Reply:
For me, the tests fail sometimes without this diff, and pass every time with it.

I'll open an issue and look at it.

Reply:
I've opened an issue here https://github.com/WordPress/gutenberg/issues/8367

Reply:
Also, updated the comment.